config/application.rb

```diff
@@ -45,6 +45,9 @@ class Application < Rails::Application
     # Version of your assets, change this if you want to expire all your assets
     config.assets.version = '2.0'
 
+    config.generators do |g|
+      g.test_framework :rspec
```

Comment:
rspec runs the tests, I probably just got that line from documentation somewhere. https://github.com/metamaps/metamaps_gen002/blob/develop/doc/RspecTesting.md explains how to run the tests - it just means executing `rspec` in the top level directory.